ios/Podfile.lock

```diff
@@ -380,15 +382,14 @@ PODS:
     - React
   - RNReactNativeHapticFeedback (1.9.0):
     - React
-  - RNReanimated (2.3.3):
+  - RNReanimated (2.9.1):
     - DoubleConversion
     - FBLazyVector
     - FBReactNativeSpec
     - glog
     - RCT-Folly
     - RCTRequired
     - RCTTypeSafety
-    - React
```

Comment:
Why is `React` removed from `Podfile.lock` ?

Reply:
It's autogenerated and React is removed as a dep from re-animated lib and not the whole project soo this should be fine 👍🏻

Reply:
thanks @siddarthkay :)